content/docs/tasks/traffic-management/custom-gateway/index.md

```diff
@@ -0,0 +1,372 @@
+    cert      1     Wed Oct 24 14:08:36 2018 DEPLOYED cert-manager-v0.6.0-dev.2 v0.6.0-dev.2  istio-system
+
+    {{< /text >}}
+
+1. To create the cluster's issuer, apply the following configuration:
+
+    {{< warning_icon >}} Change the cluster's issuer provider with your own configuration values. The example uses the values under `route53`.
+
+    {{< text yaml >}}
+
+    apiVersion: certmanager.k8s.io/v1alpha1
+    kind: ClusterIssuer
+    metadata:
+      name: letsencrypt-demo
+      namespace: kube-system
+    spec:
+      acme:
+        # The ACME server URL
+        server: https://acme-v02.api.letsencrypt.org/directory
+        # Email address used for ACME registration
+        email: <REDACTED>
+        # Name of a secret used to store the ACME account private key
+        privateKeySecretRef:
+          name: letsencrypt-demo
+        dns01:
+          # Here we define a list of DNS-01 providers that can solve DNS challenges
+          providers:
+          - name: aws-dns
+            route53:
+              accessKeyID: <REDACTED>
+              region: eu-central-1
+              secretAccessKeySecretRef:
+                name: prod-route53-credentials-secret
+                key: secret-access-key
+
+    {{< /text >}}
+
+1. If you use the `route53` provider, you must provide a secret to perform DNS ACME Validation. To create the secret, apply the following configuration file:
+
+    {{< text yaml >}}
+
+    apiVersion: v1
+    kind: Secret
+    metadata:
+      name: prod-route53-credentials-secret
+    type: Opaque
+    data:
+      secret-access-key: <REDACTED BASE64>
+
+    {{< /text >}}
+
+1. Create your own certificate:
+
+    {{< text yaml >}}
+
+    apiVersion: certmanager.k8s.io/v1alpha1
+    kind: Certificate
+    metadata:
+      name: demo-certificate
+      namespace: istio-system
+    spec:
+      acme:
+        config:
+        - dns01:
+            provider: aws-dns
+          domains:
+          - '*.mydemo.com'
+      commonName: '*.mydemo.com'
+      dnsNames:
+      - '*.mydemo.com'
+      issuerRef:
+        kind: ClusterIssuer
+        name: letsencrypt-demo
+      secretName: istio-customingressgateway-certs
+
+    {{< /text >}}
+
+    Make a note of the value of `secretName` since a future step requires it.
+
+1. To scale automatically, declare a new horizontal pod autoscaler with the following configuration:
+
+    {{< text yaml >}}
+
+    apiVersion: autoscaling/v1
+    kind: HorizontalPodAutoscaler
+    metadata:
+      name: istio-internal-custom
+      namespace: istio-system
+    spec:
+      maxReplicas: 5
+      minReplicas: 1
+      scaleTargetRef:
+        apiVersion: apps/v1beta1
+        kind: Deployment
+        name: istio-internal-custom
+      targetCPUUtilizationPercentage: 80
+    status:
+      currentCPUUtilizationPercentage: 0
+      currentReplicas: 1
+      desiredReplicas: 1
+
+    {{< /text >}}
+
+1. Apply your deployment with declaration bellow.
```

Comment:
Why isn't this done with helm? For example: https://preliminary.istio.io/docs/examples/advanced-gateways/egress-gateway-mtls-origination/#redeploy-the-egress-gateway-with-the-client-certificates

Reply:
The purpose of my initial post was to explain how to configure a custom gateway with example, to understand how it's works

Reply:
I agree with @frankbu, I would use helm as much as possible. See for example https://preliminary.istio.io/docs/tasks/traffic-management/secure-ingress/#redeploy-istio-ingressgateway-with-the-new-certificates, since this is the recommended way to deploy Istio. The focus of the task should be on cert-manager and on the point that you can have multiple ingress gateways.